statements/mandatory/completed.md

```diff
@@ -0,0 +1,65 @@
+# Completed
+
+## Description
+
+The user has successfully terminated its assessment.
+The user has answered all the questions necessary for an evaluation. 
+The assessment is sent to evaluation or has been evaluated but the user can no longer attempt the assessment until being allowed by the assessment administrator.
+
+## Examples
+
+```json
+{
+   "actor": {
+      "account": {
+         "name": "john",
+         "homePage": "http://gaiax.org"
+      }
+   },
+   "verb": {
+      "id": "http://adlnet.gov/expapi/verbs/completed"
+   },
+   "object": {
+      "id": "http://gaiax.org/xapi/activities/12d4f3f2-58e0-4a42-b3c6-b9b9cd7b0a7c",
+      "definition": {
+         "type": "http://adlnet.gov/expapi/activities/assessment",
+         "name": {
+            "en": "How much do you know about xAPI?"
+         }
+      }
+   },
+   "context": {
+      "contextActivities": {
+         "category": [
+            {
+               "id": "https://w3id.org/xapi/assessment",
+               "definition": {
+                  "type": "http://adlnet.gov/expapi/activities/profile"
+               }
+            }
+         ]
+      }
+   },
+   "result": {
+      "success": true,
+      "duration": "PT16.36S"
+   },
+   "timestamp": "2016-06-09T15:34:26.887Z"
+}
+```
+
+## Determining properties
+
+| Property  | Value         |
+|----------------|-----------------|
+| verb.id | Must be `http://adlnet.gov/expapi/verbs/completed` |
+| object.definition.type | Must be `http://adlnet.gov/expapi/activities/assessment` |
+
+## Rules
+
+- `context.contextActivities.category`: INCLUDED, MUST contain an activity with the `https://w3id.org/xapi/assessment` id.
+- `result.score`: EXCLUDED.
+- `result.success`: EXCLUDED.
```

Comment:
sure?

Reply:
It is the responsibility of passed and failed statements to carry this information